app/client/components/contactUs/contactUs.tsx

```diff
@@ -233,156 +246,166 @@ export const ContactUs = (props: ContactUsProps) => {
 
   return (
     <>
-      <ContactUsHeader />
-      <ContactUsPageContainer>
-        <div
+      <div
```

Comment:
This seems like a huge change but it was just a move of `<ContactUsHeader />` and `<ContactUsPageContainer>` from here into `ContactUsPage ` that caused a re-identation of the code inside this return. It apparently confused GitHub in the comparison.